recipes-core/packagegroups/packagegroup-ni-base.bb

```diff
@@ -21,7 +21,6 @@ ALL_DISTRO_ARM_PACKAGES = "\
 
 ALL_DISTRO_x64_PACKAGES = "\
 	linux-firmware-i915 \
```

Comment:
I wonder if we should do this change for the i915 firmware too. It shouldn't be needed for the text console in safemode.

Reply:
Yeah, I wondered about that especially because the i915 firmware adds 10Mb to the safemode (10Mb more than the sumo safemode). But I wasn't sure what feature the firmware enabled. What does it do?

Reply:
It's the GPU firmware for i915 based controllers (cRIO-903x, 904x). I guess we should make sure that the safemode text console works w/o it if you connect a monitor - I expect it to continue working with the generic VGA driver.

Reply:
I've created a work [item](https://ni.visualstudio.com/DevCentral/_workitems/edit/1745937) to look into this.